lib/index.js

```diff
@@ -19,8 +19,36 @@ var hasValidApplyPoint = function (policy) {
     return !policy.applyPoint || _applyPoints.indexOf(policy.applyPoint) !== -1;
 };
 
+// Memoizes by converting ['policy1', ..., 'policyN'] to 'policy1,...,policyN' as key
+var determineAggregateApplyPoint = _.memoize(function (policyNames) {
+
+    Hoek.assert(Array.isArray(policyNames), 'Requires array of policy names.');
+    Hoek.assert(policyNames.length > 0, 'Requires non-empty array of policy names.');
+    Hoek.assert(_.intersection(data.names, policyNames).length === policyNames.length, 'Requires loaded policy names.');
+
+    var firstPolicy = policyNames[0];
+
+    var applyPoint;
+    for (var i = 0; i < _applyPoints.length; ++i) {
+
+        // Check if the first policy appears to have this apply point.
+        if (!applyPoint &&
+            Object.keys(data[_applyPoints[i]]).indexOf(firstPolicy) !== -1) {
+
+            applyPoint = _applyPoints[i];
+        }
```

Comment:
The `break` was giving me hell in code coverage, since `i < _applyPoints.length` was always `true`.  I figured, the code isn't hot (since it's memoized), `_applyPoints` is a fixed length, and the performance concerns are minimal anyway, so I left it like this.  We could alternately test `MrHorse.parallel` with some `onPreResponse` policies and add the `break` in.  But it's clear that the order of `_applyPoints` shouldn't matter at all.